test/smoke_test/bcda_client.go

```diff
@@ -99,70 +100,132 @@ func startJob(endpoint, resourceType string) *http.Response {
 
 	req.Header.Add("Prefer", "respond-async")
 	req.Header.Add("Accept", "application/fhir+json")
-	req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", accessToken))
 
-	resp, err := client.Do(req)
+	resp, err := c.Do(req)
 	if err != nil {
-		panic(err)
+		return "", err
 	}
+	defer resp.Body.Close()
 
-	return resp
+	switch resp.StatusCode {
```

Comment:
Since this switch is only handling two cases (accepted and default) is it worth keeping it like this or could it simply be an if else.